src/overlays/actors/ovl_En_Cow/z_en_cow.c

```diff
@@ -57,9 +58,9 @@ static ColliderCylinderInit sCylinderInit = {
     { 30, 40, 0, { 0, 0, 0 } },
 };
 
-static Vec3f D_809E010C = { 0.0f, -1300.0f, 1100.0f };
+static Vec3f headFocusOffset = { 0.0f, -1300.0f, 1100.0f };
```

Comment:
```suggestion
static Vec3f sHeadFocusOffset = { 0.0f, -1300.0f, 1100.0f };
```
we s-prefix symbol names that are used in a single file
and g-prefix the others
(not directly related to using `static`, many s-prefixed symbols are not static just so they appear in the map file build/z64.map)